### PR TITLE
LPS 131380 Sidebar items are cut off the screen on the bottom

### PR DESCRIPTION
**Pull request by EliveltonR18**

### CODE REVIEW

File: modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/sidebar/Sidebar.scss [hunk 1]
```diff
@@ -5,7 +5,7 @@
 		box-shadow: none;
 
 		.sidebar-body {
-			height: calc(100vh - 224px);
+			height: calc(100vh - 273px);
```
Comment (interaminense) [inline modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/sidebar/Sidebar.scss:8]: Hi @EliveltonR18, The difference between 273 and 224 is 13px, but I think you can use our baseline of 16px here

**Review by interaminense — CHANGES_REQUESTED** [review]

File: modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/sidebar/Sidebar.scss (hunk 1 above)
Comment (EliveltonR18) [reply modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/sidebar/Sidebar.scss:8]: Hi @interaminense, I did the change, but the sidebar yet be cutted.
Screenshot from 2021-05-03 11-46-45

**Comment by EliveltonR18** [comment]
ci:test:relevant

File: modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/sidebar/Sidebar.scss (hunk 1 above)
Comment (EliveltonR18) [reply modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/sidebar/Sidebar.scss]: @interaminense I changed the height to 272px, I think it's ok now.

**Comment by EliveltonR18** [comment]
ci:test:sf

**Review by interaminense — APPROVED** [review]
LGTM!

**Comment by interaminense** [comment]
ci:test:relevant

**Comment by interaminense** [comment]
ci:forward

**Pull request closed without merging**